src/main/webapp/app/shared/organization-selector/organization-selector.component.spec.ts

```
@@ -40,38 +43,64 @@ describe('OrganizationSelectorComponent', () => {
-                organizationService = TestBed.inject(OrganizationManagementService);
-            });
+            .compileComponents();
+
+        fixture = TestBed.createComponent(OrganizationSelectorComponent);
+        component = fixture.componentInstance;
+        organizationService = TestBed.inject(OrganizationManagementService);
     });
 
     beforeEach(() => {
         jest.clearAllMocks();
     });
```

Biome (2.4.4)

[error] Duplicate beforeEach hook found.

(lint/suspicious/noDuplicateTestHooks)